src/FlashMessages.php

```diff
@@ -92,12 +92,17 @@ public function flash(string $key, $value, int $hops = 1): void
      * using this method, you may make the value available during the current
      * request as well.
      *
+     * If you want the value to be visible only in the current request, you may
+     * pass zero as the third argument.
+     *
      * @param mixed $value
      */
     public function flashNow(string $key, $value, int $hops = 1): void
     {
         $this->currentMessages[$key] = $value;
-        $this->flash($key, $value, $hops);
+        if ($hops > 0) {
+            $this->flash($key, $value, $hops);
+        }
```

Comment:
I think a test should be added that the message is not added to the session if the value for hops is `0`.

Reply:
Good catch. Done.

Reply:
Sorry, I mean a new and separated test for this. The previous one was good.

Maybe:
```php
public function testFlashNowWithZeroHopsShouldNotSetValueToSession()
{
    $this->session
        ->expects($this->never())
        ->method('set');

    $flash = FlashMessages::createFromSession($this->session);
    $flash->flashNow('test', 'value', 0);
}
```

Reply:
Done.